prototype2/server/src/engine/io/event_distribution.rs

```diff
@@ -0,0 +1,50 @@
+use specs;
+use engine;
+
+use common::protocol::ClientPayload;
+use engine::io::health_check::HealthyEvent;
+use engine::player::connection::ConnectEvent;
+use engine::player::snapshot::SnapshotAckEvent;
+use engine::player::input::InputEvent;
+
+use itertools::Itertools;
+
+/**
```

Comment:
Do you have an idea if you plan to use rustdoc stuff at all in the future?
ie. using the `///` and `//!`
Apparently the style guide wants you to use `//` instead of block comments
https://github.com/rust-lang/rust/blob/master/src/doc/style/style/comments.md